apps/web/src/pages/Applications/ApplicationSelfDeclarationPage/SelfDeclaration/CommunitySelection.tsx

```diff
@@ -54,238 +43,132 @@ export const CommunityList = ({ labels }: CommunityListProps) => {
+          <Checklist
+            idPrefix="firstNations"
+            id="firstNations"
+            name="communities"
+            legend={labels.firstNations}
+            trackUnsaved={false}
+            items={[
+              {
+                value: "status",
+                label: intl.formatMessage({
+                  defaultMessage: '"I am Status First Nations"',
+                  id: "ssJxrj",
+                  description:
+                    "Text for the option to self-declare as a status first nations",
+                }),
+              },
+              {
+                value: "nonStatus",
+                label: intl.formatMessage({
+                  defaultMessage: '"I am Non-Status First Nations"',
+                  id: "sSE4kt",
+                  description:
+                    "Text for the option to self-declare as a non-status first nations",
+                }),
+              },
+            ]}
+          />
+        </div>
+        <div data-h2-flex-item="base(1of4)" data-h2-align-self="base(center)">
+          <CommunityIcon
+            values={["status", "nonStatus"]}
+            community="first-nations"
+          />
+        </div>
+        <div data-h2-flex-item="base(3of4)" data-h2-align-self="base(center)">
+          <Checklist
+            idPrefix="inuk"
+            id="inuk"
+            name="communities"
+            legend={labels.inuk}
+            trackUnsaved={false}
+            items={[
+              {
+                value: "inuk",
+                label: intl.formatMessage({
+                  defaultMessage: `"I am Inuk"`,
+                  id: "vDb+O+",
+                  description: "Label text for Inuk community declaration",
+                }),
+              },
+            ]}
+          />
+        </div>
+        <div data-h2-flex-item="base(1of4)" data-h2-align-self="base(center)">
+          <CommunityIcon values={["inuk"]} community="inuit" />
+        </div>
+        <div data-h2-flex-item="base(3of4)" data-h2-align-self="base(center)">
+          <Checklist
+            idPrefix="metis"
+            id="metis"
+            name="communities"
+            legend={labels.metis}
+            trackUnsaved={false}
+            items={[
+              {
+                value: "metis",
+                label: intl.formatMessage({
+                  defaultMessage: `"I am Métis"`,
+                  id: "/81xCT",
+                  description: "Label text for Métis community declaration",
+                }),
+              },
+            ]}
+          />
+        </div>
+        <div data-h2-flex-item="base(1of4)" data-h2-align-self="base(center)">
+          <CommunityIcon values={["metis"]} community="metis" />
+        </div>
+        <div data-h2-flex-item="base(3of4)" data-h2-align-self="base(center)">
+          <Checklist
+            idPrefix="other"
+            id="other"
+            name="communities"
+            legend={labels.other}
```

Comment:
I think the label should be "My community isn't listed"
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/0a891dde-42e3-470a-bfc9-5337f22ea26f)

Reply:
Yes, it should be "My community isn't listed".

Reply:
Oops, updated with [ee6e8e4](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7140/commits/ee6e8e4d26cf743a97c236d6a8b4b7fda8f7f518).